libraries/AP_ServoRelayEvents/AP_ServoRelayEvents.cpp

```diff
@@ -102,13 +103,17 @@ void AP_ServoRelayEvents::update_events(void)
         return;
     }
 
+    if (channel > NUM_SERVO_CHANNELS || channel == 0) {
+        return;
+    }
+
     start_time_ms = AP_HAL::millis();
 
     switch (type) {
     case EVENT_TYPE_SERVO:
         hal.rcout->enable_ch(channel-1);
         if (repeat & 1) {
-            hal.rcout->write(channel-1, RC_Channel::rc_channel(channel-1)->get_radio_trim());
```

Comment:
old code wrote the trim value out , new code writes the current pwm value.   not sure if it's important, but seems like a logical difference in the code.